streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -427,8 +427,18 @@
 
     /** {@code max.task.idle.ms} */
     public static final String MAX_TASK_IDLE_MS_CONFIG = "max.task.idle.ms";
-    private static final String MAX_TASK_IDLE_MS_DOC = "Maximum amount of time in milliseconds a stream task will stay idle when not all of its partition buffers contain records," +
-        " to avoid potential out-of-order record processing across multiple input streams.";
+    private static final String MAX_TASK_IDLE_MS_DOC = "This config controls whether joins and merges"
+        + " may produce out-of-order results."
```

Comment:
Are these leading spaces epected?

Reply:
Yep. This is just string concatenation, so we need to add a space between "merges" and "may". I could have put them at the end, but the beginning makes it easier to spot if we're missing one.